tests/drivers/i2c/i2c_sensor/testcase.yaml

```
@@ -0,0 +1,6 @@
+common:
+  tags: drivers i2c
+  depends_on: i2c
+tests:
+  drivers.i2c.controller_to_sensor:
```

This should reflect the test location, e.g.:

```suggestion
  drivers.i2c.sensor:
```